src/internal/modules/reporting/controllers.js

```diff
@@ -0,0 +1,22 @@
+const services = require('../../lib/connectors/services');
+
+const getChargingForecastReportsPage = (request, h) => {
+  return h.view('nunjucks/reporting/charging-forecast-reports', {
+    ...request.view,
+    back: '/manage',
+    pageTitle: 'Download a charging forecast report'
+  });
+};
+
+const getDownloadableReport = async (request, h) => {
+  const { reportIdentifier } = request.params;
```

Comment:
If the water service is responding with a CSV file, we should be able to pipe it directly through.